### PR TITLE
chore(swatch-picker): add bundle size configuration

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
Added bundle size configuration for the SwatchPicker

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports | Baseline (minified/GZIP) | PR    | Change     |
| :---------------- | -----------------------: | ----: | ---------: |
| <samp>react-swatch-picker-preview</samp> <br /> <abbr title='bundle-size/index.fixture.js'>@fluentui/react-swatch-picker-preview - package</abbr> | `0 B`<br />`0 B` | `103.385 kB`<br />`29.763 kB` | 🆕 New entry|

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/fa0be6273745fe22574c29db093d64db12348a30'>fa0be6273745fe22574c29db093d64db12348a30</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-swatch-picker-preview-6cb1ebfe-d839-4335-adb5-483fc5a0ed27.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (Hotell) [inline change/@fluentui-react-swatch-picker-preview-6cb1ebfe-d839-4335-adb5-483fc5a0ed27.json:2]: this has no effect on runtime that we ship to npm thus  none

```suggestion
  "type": "none",
```

File: change/@fluentui-react-swatch-picker-preview-6cb1ebfe-d839-4335-adb5-483fc5a0ed27.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "comment": "fix(swatch-picker): added bundle size configuration",
+  "packageName": "@fluentui/react-swatch-picker-preview",
+  "email": "vkozlova@microsoft.com",
+  "dependentChangeType": "patch"
```
Comment (Hotell) [inline change/@fluentui-react-swatch-picker-preview-6cb1ebfe-d839-4335-adb5-483fc5a0ed27.json:6]:
```suggestion
  "dependentChangeType": "none"
```

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by Hotell — APPROVED** [review]

**Pull request merged**